pipeline/vars/common.groovy

```diff
@@ -38,7 +38,6 @@ def runTestSuite() {
     cmd += " --instances-name ${instanceName}"
 
     // Append test suite specific
-    cmd += " --inventory ${env.sutVMConf}"
```

Comment:
IMO, we should ensure the value provided from the stage has the highest precedence. For example
```
def sutVMConf = "${env.sutVMConf}" ?: ""

... <inside CI_message> if statement
  if (! sutVMConf?.trim()) {
    if (env.composeId.indexOf('RHEL-8') >= 
    ...
  }

... in the last
if (sutVMConf?.trim()) {
  cmd += " --inventory ${sutVMConf}"
}
```

Reply:
I have made the necessary changes.